src/dune_engine/scheduler.ml

```diff
@@ -1067,7 +972,11 @@ module Run = struct
             loop ()
           | Ok `Stop -> Fiber.return ()))
     in
-    let run = Rpc0.with_rpc_serve t.rpc loop in
+    let run =
+      match once with
+      | None -> loop
+      | Some once -> fun () -> Fiber.fork_and_join_unit once loop
```

Comment:
What does `once` mean now? I kind of expected it meant `init`, but if it's for initialisation it should be done before `loop`, not in parallel. Is it still used BTW? It seems that we don't use this argument anymore in the `bin/` directory.

Reply:
Good call. Let me give that a try.

Reply:
Actually, this isn't going to be clean because `poll` and `go` initialize the scheduler differently. For example, the `on_event` callback for `poll` and `go` is different.

Reply:
Does it need to be different?

Reply:
The `polling` field is unused btw